spec/requests/api/v1/sessions/create_spec.rb

```diff
@@ -0,0 +1,87 @@
+RSpec.describe 'POST /api/v1/users/sign_in', type: :request do
+  subject { post user_session_url, params: user_params, as: :json }
+
+  context 'when the request is valid' do
+    let(:user) { create(:user) }
+    let(:user_params) do
+      {
+        'user':
+          {
+            'email': user.email,
+            'password': user.password
+          }
+      }
+    end
+
+    it 'returns the user' do
+      subject
+      expect(response.body).to include_json(
+        user: {
+          id: user.id,
+          email: user.email,
+          provider: user.provider,
+          full_name: user.full_name,
+          uid: user.uid
+        }
+      )
+    end
+
+    it 'returns a success status' do
+      subject
+      expect(response).to be_successful
+    end
+
+    it 'returns the user with token in the header' do
+      subject
+      expect(response.header['access-token']).to be_present
+    end
+  end
+
+  context 'when the user not exists' do
+    let(:user_params) do
+      {
+        user:
+          {
+            email: 'no_user@mail',
+            password: 'no_user'
+          }
+      }
+    end
+
+    it 'returns the error message' do
+      subject
+      error_msg = 'Invalid login credentials. Please try again.'
```

Comment:
maybe you can move this error to ```en.yml``` and call it ```I18N.t('api.tests...')```. WDYT?

Reply:
WDYT about adding this gem? [i18n_linter](https://github.com/rootstrap/i18n_linter)